speculation-rules/prerender/service-workers.https.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<title>Same-origin prerendering can access Indexed Database</title>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="resources/utils.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<body>
+<script>
+
+setup(() => assertSpeculationRulesIsSupported());
+
+const ACTIVATION_DELAY = 500;
+
+promise_test(async t => {
+  const reg = await service_worker_unregister_and_register(
+    t, "./resources/service-worker.js", `resources/`);
+
+  t.add_cleanup(() => reg.unregister());
+  const {exec} = await create_prerendered_page(t);
+  const {text, prerendering} = await exec(async () => {
+    const text = await (await fetch(`ping`)).text();
+    return {text, prerendering: document.prerendering};
+  });
+
+  assert_true(prerendering);
+  assert_equals(text, 'pong');
+}, 'A prerendered page should be able to access an existing Service Worker');
+
+promise_test(async t => {
+  const {exec, activate} = await create_prerendered_page(t);
+  const scope = `/${token()}/`;
+  await exec(async scope => {
+    window.serviceWorkerInstalled = new Promise(resolve => {
+      navigator.serviceWorker.register('./service-worker.js', {scope: '.' + scope})
```

Comment:
nit: why not include this `.` when `scope` is defined above?

Reply:
sure